lib/storage/providers/SQLiteStorage.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * The SQLiteStorage provider stores everything in a key/value store by
+ * converting the value to a JSON string
+ */
+import {open} from 'react-native-quick-sqlite';
+import _ from 'underscore';
+
+const DB_NAME = 'Expensify-new-db';
+const db = open({name: DB_NAME});
+
+db.execute('CREATE TABLE IF NOT EXISTS magic_map (record_key TEXT NOT NULL PRIMARY KEY , valueJSON JSON NOT NULL) WITHOUT ROWID;');
+
+// All of the 3 pragmas below were suggested by SQLite team.
+// You can find more info about them here: https://www.sqlite.org/pragma.html
+db.execute('PRAGMA CACHE_SIZE=-20000;');
+db.execute('PRAGMA synchronous=NORMAL;');
+db.execute('PRAGMA journal_mode=WAL;');
+
+const provider = {
+    /**
+    * Get the value of a given key or return `null` if it's not available in storage
+    * @param {String} key
+    * @return {Promise<*>}
+    */
+    getItem(key) {
+        return db.executeAsync('SELECT record_key, valueJSON FROM magic_map WHERE record_key=?;', [key]).then(({rows}) => {
```

Comment:
For consistency with other code we have, let's format all queries to have spaces before/after the comparisons. ie:
```suggestion
        return db.executeAsync('SELECT record_key, valueJSON FROM magic_map WHERE record_key = ?;', [key]).then(({rows}) => {
```